rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -155,6 +168,10 @@ class NodeBase : public NodeBaseInterface, public std::enable_shared_from_this<N
   mutable std::recursive_mutex notify_guard_condition_mutex_;
   rclcpp::GuardCondition notify_guard_condition_;
   bool notify_guard_condition_is_valid_;
+  rclcpp::executors::SingleThreadedExecutor::SharedPtr executor_;
```

Comment:
Add some comments or insert empty line (These variables are for different functionality)